bootstrap/components/sinks/v002__druid-sink-topology-component.json

```diff
@@ -0,0 +1,352 @@
+                  "tooltip": "fieldName",
+                  "isOptional": false,
+                  "type": "enumstring",
+                  "options": [],
+                  "hint": "inputFields"
+                }
+             ]
+           },
+           {
+             "uiName": "Long Sum Aggregator",
+             "fieldName": "longsum",
+             "tooltip": "Long sum Aggregator",
+             "isOptional": true,
+             "type": "object",
+             "defaultValue": null,
+             "fields": [
+                {
+                  "uiName":"name",
+                  "fieldName": "name",
+                  "tooltip": "counter name",
+                  "isOptional": false,
+                  "type": "string",
+                  "defaultValue": null
+                },
+                {
+                  "uiName":"fieldName",
+                  "fieldName": "fieldName",
+                  "tooltip": "fieldName",
+                  "isOptional": false,
+                  "type": "enumstring",
+                  "options": [],
+                  "hint": "inputFields"
+                }
+             ]
+           },
+           {
+             "uiName": "Long Max Aggregator",
+             "fieldName": "longmax",
+             "tooltip": "Long max Aggregator",
+             "isOptional": true,
+             "type": "object",
+             "defaultValue": null,
+             "fields": [
+                {
+                  "uiName":"name",
+                  "fieldName": "name",
+                  "tooltip": "counter name",
+                  "isOptional": false,
+                  "type": "string",
+                  "defaultValue": null
+                },
+                {
+                  "uiName":"fieldName",
+                  "fieldName": "fieldName",
+                  "tooltip": "fieldName",
+                  "isOptional": false,
+                  "type": "enumstring",
+                  "options": [],
+                  "hint": "inputFields"
+                }
+             ]
+           },
+           {
+             "uiName": "Long Min Aggregator",
+             "fieldName": "longmin",
+             "tooltip": "Long min Aggregator",
+             "isOptional": true,
+             "type": "object",
+             "defaultValue": null,
+             "fields": [
+                {
+                  "uiName":"name",
+                  "fieldName": "name",
+                  "tooltip": "counter name",
+                  "isOptional": false,
+                  "type": "string",
+                  "defaultValue": null
+                },
+                {
+                  "uiName":"fieldName",
+                  "fieldName": "fieldName",
+                  "tooltip": "fieldName",
+                  "isOptional": false,
+                  "type": "enumstring",
+                  "options": [],
+                  "hint": "inputFields"
+                },
+               {
+                 "uiName": "Parallelism",
```

Comment:
This does not seem to be at the right level. Can you double check if it translates to the correct number of executors in the underlying storm topology? Same for v001 json.